src/graphql/Raffle/Mutation.ts

```diff
@@ -0,0 +1,137 @@
+import { User, Context, generateToken, Auth, isAuth} from '../../utils'
+import bcrypt from 'bcrypt'
+
+/**
+ * this function create user
+ * @param parent 
+ * @param args 
+ * @param ctx 
+ */
+async function createUser(parent: object, args: User, ctx: Context): Promise<User> {
+    const {prisma, request} = ctx
+    const {data} = args
+    
+    isAuth(request)
+
+    try {
+        const userCreated =  await prisma.users.create({
+            data
+        })
```

Comment:
Is an standard use tabs on 2 spaces for JS and TS. 

Can't you change that on your linter config please?

Reply:
ok